src/main/scala/edu/berkeley/ce/rockslicing/CommandReader.scala

```diff
@@ -0,0 +1,13 @@
+package edu.berkeley.ce.rockslicing
+
+/** A simple tool for parsing command line inputs
+  * @constructor Create a new command reader
+  */
+case class CommandReader(inputs: Array[String]) {
```

Comment:
Okay, I'll try to follow standard conventions. Haven't coded command line arguments before, I'll see what I can find online regarding this. Do you have any guidance or references that I should check out?